index.html

```diff
@@ -7,62 +7,64 @@
     <title>VEOHRC Third Party Widget</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <!--
-    	need to set up server to use:
-    	X-Frame-Options: allow-from https://example.com/
-    -->
-    <!-- <script src='https://www.google.com/recaptcha/api.js?render=6LfGRX0UAAAAAINiXRMozu84LiFGNjdKn-QRwgr4'></script> -->
 </head>
 <body>
-	<main>
-		<header>
-			<h1>Have you been treated unfairly in Victoria?</h1>
-			<p class="intro">Report it securely and confidentially to the Victorian Equal Opportunity & Human Rights Commission.</p>
-		</header>
+	<div>
+		<h3>Have you been treated unfairly in Victoria?</h3>
+		<p class="form-description">
+			Report it securely and confidentially to the Victorian Equal Opportunity & Human Rights Commission.
+		</p>
+	</div>
+	<div>
 		<form id="contact" method="post" >
-			<fieldset id="collapse">
-				<label><span class="label">Your name</span><input id="name" name="name" minlength="2" type="text" required></label>
-				<label><span class="label">Your email address</span><input required type="email" id="email" name="email" value="" placeholder=""></label>
-				<!-- insert Phone number (v2) -->
-				<label for="what">What happened?</label>
-				<p class="hidden what_info">Describe what happened, and where it happened. Just a sentence or two is fine!</p>
-				<textarea required name="what_happened" id="what"></textarea>
-				<label><span class="label">Where was it? <small>(Postcode)</small></span><input minlength="4" maxlength="4"  required type="text" id="postcode" name="postcode" value="" placeholder=""></label>
-				<!-- insert Postcode field. consider http://v0.postcodeapi.com.au/suburbs/?q=****&state=vic api
-					or http://api.geonames.org/postalCodeLookupJSON?formatted=true&placename=*****&country=AU&username=demo&style=full&adminCode1=VIC
-				-->
-			</fieldset>
-			<fieldset class="showconf">
-				<label class="checkbox">
-					<input type="checkbox" name="dont_contact_me" id="nocontact" />
-					<small>I do not wish to be contacted</small>
+			<label  for="name" class="label h4">Your name (required)</label>
+			<input name="name" id="name" minlength="2" type="text" required>
+			
+			<label for="email" class="label h4">Your email address (required)</label>
+			<input name="email" required type="email" id="email" value="" placeholder="">
+			
+			<label for="what" class="label h4">What happened? (required)</label>
+			<textarea
+				name="what_happened"
+				id="what"
+				required placeholder="Describe what happened, and where it happened. Just a sentence or two is fine!"
+				aria-describedby="Describe what happened, and where it happened. Just a sentence or two is fine!" /></textarea>
+
+			<label for="postcode" class="label h4">Where was it? - Postcode (required)</label>
+			<input
+				name="postcode"
+				minlength="4"
+				maxlength="4"
+				required
+				type="text"
```

Comment:
I was going to suggest type="number" but that generally includes a stepper in the UI right? Probably doesn't make sense here. Is there a drawback to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-pattern ?

Reply:
I tried the pattern attribute but it didn't through any validation errors. I have added some extra validation to the postcode field. I have also removed the maxLength which is meant to be bad accessibility practice.